### DISCUSSION TITLE
Geometry management

### DISCUSSION THREAD

**Discussion by rodneytamblyn** [trevordevore/levure#175] [General]
Hi Trevor,

I see Levure provides standard support for managing window sizes (https://github.com/trevordevore/levure/wiki/helper-window_manager).  Just wondering what your personal approach is to doing geometry management.  I presume you're probably not using LC GM.  Are there any other features within Levure for this?

~ Rodney

**Comment by trevordevore** [comment]
@rodneytamblyn I typically code geometry by hand. In the `resizeStack` handler I reposition root level objects. If a root level object is a group then as a general rule I add the logic for resizing controls in that group in the `resizeControl` handler of the group. This simplifies the geometry code as each handler is only resizing and repositioning immediate children.
